syntax = "proto3";

package com.waitingforcode;

import "google/protobuf/timestamp.proto";

message Visit {
  string visit_id = 1;
  google.protobuf.Timestamp event_time = 2;
  string user_id = 3;
  string page = 4;
  string ip = 5;
  string login = 6;
  bool is_connected = 7;
  string from_page = 8;
}